egs/ifnenit/v1/local/ienit_initialize.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+. ./cmd.sh
+. ./path.sh
+
+# This script is originally from qatip project (http://qatsdemo.cloudapp.net/qatip/demo/)
+# of Qatar Computing Research Institute (http://qcri.qa/)
+
+data_dir="/export/b01/babak/IFN-ENIT/ifnenit_v2.0p1e"
```

Comment:
This should be set via a command line option. 
Rename the variable to `database_dir`, move the line to run.sh, and pass it to related scripts from there.